Stylesheet/Components/alert.css

```diff
@@ -1,2 +1,36 @@
 /* This css file is for alerts */
 
+.alert-box{
+    max-width: 90%;
+    padding: 0.5rem;
```

Comment:
whenever you are doing `max-width /width: X% or Xvw` add `margin; 0 auto` to center it within container

Reply:
Okay, doing that